pom.xml

```diff
@@ -276,12 +276,12 @@
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcprov-jdk15on</artifactId>
-        <version>1.66</version>
+        <version>1.68</version>
       </dependency>
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcpg-jdk15on</artifactId>
-        <version>1.66</version>
+        <version>1.68</version>
       </dependency>
```

Comment:
Just for my own edification: bcprov is the Bouncy Castle provider and bcpg is the Bouncy Castle OpenPGP API. If I'm reading the Maven repo info correctly, bcpg depends on bcprov so it doesn't seem outlandish that they'd need to be upgraded in lockstep:

https://mvnrepository.com/artifact/org.bouncycastle/bcpg-jdk15on/1.68